exist-core/src/test/xquery/numbers/format-numbers.xql

```diff
@@ -245,6 +245,14 @@ function fd:simple-number-2($number, $picture) {
     format-number($number, $picture)
 };
 
+declare
+    %test:args(12345.6)
+    %test:args("#.###,##")
+    %test:assertEquals("12345.6")
```

Comment:
I just checked: Saxon 10 (HE) does output `12345.6` as well. I still think this is not correct because the spec does not say the grouping character has to be `,` and the decimal delimiter is `.`. Isn't it the idea of a picture string to change these defaults?

Reply:
These are just the defaults as far as I understood.
Similar to the example in the spec: `format-number(1234.5678, '#ʹ##0·00', 'ch')`
If `format-number(1234.5678, '#,###.##', 'de')` is supported (Saxon 10 HE does not) I stop complaining.

Reply:
If you don't specify a specific default format, there is a *default decimal format* which is defined in the XQuery spec. Take a look at that.

Reply:
@adamretter you are right. The relevant portion in the spec is statically known decimal formats: https://www.w3.org/TR/2017/REC-xquery-31-20170321/#dt-static-decimal-formats
Therefore the defaults '.' and ',' cannot be changed without providing the third parameter.